### PR TITLE
Waiter panel page

### PR DESCRIPTION
**Pull request by andriifromlviv**
page object and page components implemented very similar to my previous pull request with AdministratorPanelPage

### CODE REVIEW

File: TAF_EasyRest/TestFramework/Pages/WaiterPanelPage.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public class WaiterPanelPage : BasePage
+    {
+        public NavigationMenuPageComponent NavigationMenuPageComponent { get; }
+        public UserMenuHeaderButtonPageComponent UserMenuHeaderButton { get; }
+
+
+        public WaiterPanelPage(IWebDriver driver) : base(driver)
+        {
+            NavigationMenuPageComponent = new NavigationMenuPageComponent(driver);
+            UserMenuHeaderButtonPageComponent UserMenuHeaderButton = new UserMenuHeaderButtonPageComponent(driver);
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/Pages/WaiterPanelPage.cs:15]: I believe you've used ctrl+c and ctrl+v, this is an anti-pattern.

**Review by LoveMyDaughter — APPROVED** [review]
Little remark:

**Review by dmatsev — APPROVED** [review]

**Review by LoveMyDaughter — APPROVED** [review]
Ok.

File: TAF_EasyRest/TestFramework/PageComponents/Administrator/AcceptedCardPageComponent.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class AcceptedCardPageComponent
+    {
+        private IWebElement _card { get; }
+
+        public AcceptedCardPageComponent(IWebElement _card)
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/PageComponents/Administrator/AcceptedCardPageComponent.cs:8]: Parameters should begin with lower case letters. Have you changed your code convention?

**Review by dmatsev — APPROVED** [review]

**Review by LoveMyDaughter — APPROVED** [review]

**Pull request merged**